apps/console/src/features/users/components/user-profile.tsx

```diff
@@ -207,9 +214,17 @@ export const UserProfile: FunctionComponent<UserProfilePropsInterface> = (
             if (schema.name !== "roles.default") {
                 if (values.get(schema.name) !== undefined && values.get(schema.name).toString() !== undefined) {
                     if (schemaNames.length === 1) {
-                        opValue = schemaNames[0] === "emails"
-                            ? { emails: [values.get(schema.name)] }
-                            : { [schemaNames[0]]: values.get(schemaNames[0]) };
+                        if (schema.extended) {
+                            opValue = {
+                                [ProfileConstants.SCIM2_ENT_USER_SCHEMA]: {
+                                    [schemaNames[0]]: values.get(schemaNames[0])
+                                }
+                            }
+                        } else {
+                            opValue = schemaNames[0] === "emails"
```

Comment:
Shouldn't the "emails" term moved to a constant?

Reply:
Fixed with d523fb9